### ISSUE TITLE
Auto Heavy Sieve Crash

### ISSUE THREAD

**Issue by Ikarsu** [BloodWorkXGaming/ExNihiloCreatio#240]
When using a mechanical user from Extra Utilities feeding Chorus Fruit to give the Auto Heavy Sieve a speed boost while sifting compressed blocks, it crashes the internal server.

Under Normal Circumstances, it gives a x4 speed boost for 10 seconds when fed by hand.

Mod version: 1.12.2-0.4.3.1
Supporting mods: Extra Utilities 1.12-1.9.9
Modpack used for setup: FTB Presents - Stoneblock II

**Comment by papaworld** [comment]
Hi Ikarsu. I'm not part of the Ex Niholo team... just a user of the mod. I experienced the issue with chorus fruit with the mechanical user and modular routers activator... same crash.

For background I used to use apples for x4, then magma slime for x2.5. Then dropped the whole idea of speed boosting by food and went to enchanting the meshes which gave much better results for yield and speed by the Steve-In-A-Box Sieve and Heavy Sieve.

Ex Creatio 0.4.4
ExCompressum 3.0.27
